app/scripts/keyring-controller.js

```diff
@@ -437,7 +433,7 @@ module.exports = class KeyringController extends EventEmitter {
       this.emit('newAccount', hexAccount)
       return this.setupAccounts(accounts)
     }).then(() => {
-      return this.placeSeedWords()
+      return this.placeSeedWords(this.getKeyringsByType('HD Key Tree')[0])
```

Comment:
hmm actually why are we placing the seed words at all? but sort of out of scope of this PR